app/controllers/menus_controller.rb

```diff
@@ -2,6 +2,11 @@ class MenusController < ApplicationController
 
   def show
     @restaurant = Restaurant.find(params[:restaurant_id])
-    @cart_nr = ShoppingCartItem.count
+    if session[:cart_id]
+      @cart = ShoppingCart.find(session[:cart_id])
+      @cart_nr = @cart.shopping_cart_items.count
```

Comment:
We use it in the menus show page view. But will later have it in the nav bar once that is ready.